src/components/Restaurants/RestaurantReview.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { FaSearch } from 'react-icons/fa'; 
+import './RestaurantList.css';
+
+const SearchBar = () => (
+  <header className="header">
+    <div className="searchContainer">
+      <input
+        type="text"
+        placeholder="Restaurants"
+        className="searchInput"
+        aria-label="Search for restaurants"
+      />
+      <input
+        type="text"
+        placeholder="Address, neighborhood"
+        className="locationInput"
+        aria-label="Location"
+      />
+      <button className="searchButton">
+        <FaSearch />
+      </button>
+    </div>
+  </header>
+);
+
+const Filters = () => (
+  <div className="filters">
+    <div className="filterSection">
+      <button className="button">$</button>
```

Comment:
A few notes:
- each button is of the same width, though some have more texts than others. (width: ...)
- no margin between each other (margin: ...)
- the left and right most button have some border radius on 2 corners, the other buttons don't have any border radius
- background should be white 
- each button does not have the right border (otherwise the right border and the left border will make the border appear bolder than it should be) (border-right: 0)
- rename class "button" to "filterButton"

The most left button should also have in addition to the notes above
```
border-top-left-radius: 16px
border-bottom-left-radius: 16px
```